packages/esm-patient-registration-app/src/patient-registration/patient-registration.test.tsx

```diff
@@ -177,260 +178,274 @@ const fillRequiredFields = async () => {
   fireEvent.click(genderInput);
 };
 
-describe('patient registration', () => {
-  beforeEach(() => {
-    mockedUseConfig.mockReturnValue(mockOpenmrsConfig);
-    mockedSavePatient.mockReturnValue({ data: { uuid: 'new-pt-uuid' }, ok: true });
-    mockedSaveEncounter.mockClear();
-    mockedShowToast.mockClear();
-  });
+describe('Patient Registration Component', () => {
+  describe('Registering a New Patient', () => {
```

Comment:
Hi @ayush-AI,
Could you please update the test names according to [this test naming convention](https://infinum.com/handbook/frontend/react/testing-best-practices#naming-convention)?

Reply:
done. please check